code/datums/ai/monkey/monkey_controller.dm

```diff
@@ -61,6 +62,15 @@ have ways of interacting with a specific mob and control it.
 
 	return ..() //Run parent at end
 
+// Stops sentient monkeys from being knocked over like weak dunces.
+/datum/ai_controller/monkey/on_sentience_gained()
+	..()
+	pawn.RemoveElement(/datum/element/connect_loc_behalf)
```

Comment:
If you're going to remove this manually, please provide more context to the call for the sake of cheapness. also does this even work? the element is on the controller not the monkey